.circleci/scripts/should_run_job.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 set -exu -o pipefail
 
+exit 0
```

Comment:
this is needed because:

there are two mechanisms to trigger jobs, one is config.yml and the other is should_run_job.sh.

`should_run_job.sh` here just whitelists any hit, because then all jobs are run